recipes-images/images/core-image-sdk.inc

```diff
@@ -7,7 +7,12 @@ sdk_post_process () {
         $SUDO_EXEC bash -c 'source "$0" && cd "${OECORE_TARGET_SYSROOT}/usr/src/kernel" && make scripts' $target_sdk_dir/environment-setup-@REAL_MULTIMACH_TARGET_SYS@
 
         # Set up kernel headers for building user application
-        $SUDO_EXEC bash -c 'source "$0" && cd "${OECORE_TARGET_SYSROOT}/usr/src/kernel" && make mrproper; make headers_check; make INSTALL_HDR_PATH=${OECORE_TARGET_SYSROOT}/usr headers_install' $target_sdk_dir/environment-setup-@REAL_MULTIMACH_TARGET_SYS@
+	if [ "${LINUX_GIT_BRANCH}" = "linux-5.10.y-cip" ]; then
+            $SUDO_EXEC bash -c 'source "$0" && cd "${OECORE_TARGET_SYSROOT}/usr/src/kernel" && cp include/config/kernel.release _kernel.release; cp include/generated/utsrelease.h _utsrelease.h; make prepare; mv _kernel.release include/config/kernel.release; mv _utsrelease.h include/generated/utsrelease.h; make headers_check; make INSTALL_HDR_PATH=${OECORE_TARGET_SYSROOT}/usr headers_install' $target_sdk_dir/environment-setup-@REAL_MULTIMACH_TARGET_SYS@
```

Comment:
Use this patch for both kernel-5.10.y and kernel-4.19.y because we shouldn't use make mrproper.